new-lamassu-admin/src/components/editableTable/Row.js

```diff
@@ -1,83 +1,178 @@
-import { Form, Formik, FastField, useFormikContext } from 'formik'
-import React, { useState, memo } from 'react'
+import { makeStyles } from '@material-ui/core'
+import { Field, useFormikContext } from 'formik'
+import * as R from 'ramda'
+import React, { useContext } from 'react'
 
-import { Link } from 'src/components/buttons'
+import { Link, IconButton } from 'src/components/buttons'
 import { Td, Tr } from 'src/components/fake-table/Table'
+import { Switch } from 'src/components/inputs'
+import { TL2 } from 'src/components/typography'
+import { ReactComponent as DisabledDeleteIcon } from 'src/styling/icons/action/delete/disabled.svg'
+import { ReactComponent as DeleteIcon } from 'src/styling/icons/action/delete/enabled.svg'
+import { ReactComponent as DisabledEditIcon } from 'src/styling/icons/action/edit/disabled.svg'
+import { ReactComponent as EditIcon } from 'src/styling/icons/action/edit/enabled.svg'
+import { ReactComponent as StripesSvg } from 'src/styling/icons/stripes.svg'
 
-const getField = (name, component, props = {}) => (
-  <FastField name={name} component={component} {...props} />
-)
+import TableCtx from './Context'
+import styles from './Row.styles'
 
-const ERow = memo(({ elements }) => {
-  const { values, submitForm, resetForm, errors } = useFormikContext()
-  const [editing, setEditing] = useState(false)
+const useStyles = makeStyles(styles)
 
-  const innerSave = () => {
-    submitForm()
+const ActionCol = ({ disabled, editing }) => {
+  const classes = useStyles()
+  const { values, submitForm, resetForm } = useFormikContext()
+  const {
+    editWidth,
+    onEdit,
+    enableEdit,
+    enableDelete,
+    disableRowEdit,
+    onDelete,
+    deleteWidth,
+    enableToggle,
+    onToggle,
+    toggleWidth,
+    actionColSize
```

Comment:
this is better than what I was proposing on https://github.com/lamassu/lamassu-server/pull/390, so I'll close that.